cmake/CMakeLists.txt

```diff
@@ -117,6 +119,10 @@ if(NO_PAE_SUPPORT)
     add_definitions(-DNO_PAE_SUPPORT)
 endif()
 
+if(USE_PAE_OLD_PARSER)
```

Comment:
maybe better to use `elseif` here, because you cannot use `NO_PAE_SUPPORT` _and_ `USE_PAE_OLD_PARSER` at the same time.